include/zfs_deleg.h

```diff
@@ -54,6 +54,7 @@ typedef enum {
 	ZFS_DELEG_NOTE_PROMOTE,
 	ZFS_DELEG_NOTE_RENAME,
 	ZFS_DELEG_NOTE_SEND,
+	ZFS_DELEG_NOTE_SEND_RAW,
```

Comment:
It looks like it should be alright, but let's make sure that reordering these enums doesn't result in any compatibility problems between ZFS versions.